### PR TITLE
Moving elementwise core to impl - clamp(FX Converter Refactor [21/N]) <Target: converter_reorg_elementwise>

### PR DESCRIPTION
**Pull request by borisfom**
# Description
aten::clamp refactor

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: py/torch_tensorrt/fx/converters/aten_ops_converters.py [hunk 1]
```diff
@@ -25,9 +25,15 @@
 from torch_tensorrt.fx.converters.impl import activation
 from torch_tensorrt.fx.converters.impl.elementwise import trunc_div
 from torch_tensorrt.fx.converters.impl.elementwise import rsqrt
+from torch_tensorrt.fx.converters.impl.elementwise import clamp
 
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
+
+def or_none(args, i):
```
Comment (narendasan) [inline py/torch_tensorrt/fx/converters/aten_ops_converters.py:33]: Thoughts on putting this in utils? Would it matter for acc? or would ACC always give you None?

**Review by narendasan — APPROVED** [review]
Mostly looks good to me, just pending Meta approval

**Review by narendasan — CHANGES_REQUESTED** [review]
Actually clamp tests are failing, seems like theres some issues

File: py/torch_tensorrt/fx/converters/aten_ops_converters.py (hunk 1 above)
Comment (borisfom) [reply py/torch_tensorrt/fx/converters/aten_ops_converters.py:33]: Well acc uses kwargs for optional arguments - but this can be put in utils anyway.

**Comment by narendasan** [comment]
Closing in favor of https://github.com/pytorch/TensorRT/pull/2070

**Pull request closed without merging**